### PR TITLE
Criteo doc updates

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## What do these changes do?

> Updates to the Criteo destination, including the new feature to map RudderStack events to Criteo standard events.

## Type of documentation

- [ ] New documentation
- [x] Documentation update
- [ ] Hotfix

### CODE REVIEW

File: docs/destinations/streaming-destinations/criteo.mdx [hunk 1]
```diff
@@ -22,47 +22,57 @@ To know more about the difference between cloud mode and device mode in RudderSt
...
-- **Map Specific Fields to Criteo Fields:** Enter the payload fields and the corresponding mapped fields that will be used to send the event data to Criteo.
+- **Client-side Events Filtering**: This setting lets you specify which events should be blocked or allowed to flow through to Mouseflow.
 
 <div class="infoBlock">
+For more information on this setting, refer to the <Link to="/sources/sdks/event-filtering/">Client-side Events Filtering</Link> guide.
+</div>
 
-Criteo events allow you to send any extra data about a page or a user to add more context to the events. You can set this feature in Criteo with the assistance of your Criteo Account Manager. You can then use the **Map Specific Fields to Criteo Fields** field mapping feature to send extra data through RudderStack.
+- **Mapping RudderStack events to the Criteo standard events**: Enter the **Event Name** and the corresponding [**Standard Event**]() from the dropdown. This standard event will be triggered when the mapped RudderStack event is called.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/criteo.mdx:61]: @mihir-4116 please share the Criteo standard events reference link to be added here. Is [this](https://www.criteo.com/wp-content/uploads/2018/09/CSPOneTag_v1.1.pdf) a valid reference?

File: docs/destinations/streaming-destinations/criteo.mdx [hunk 2]
```diff
@@ -22,47 +22,57 @@ To know more about the difference between cloud mode and device mode in RudderSt
...
-<img src="../../assets/Criteo.png" />
+## Connection settings
 
-### Configuration Settings
+To successfully configure Criteo as a destination, you need to configure the following settings:
 
-To successfully configure Criteo as a destination, you will need to configure the following settings:
+<img src="../../assets/event-stream-destinations/criteo-connection-settings-1.png" alt="Criteo connection settings" />
+<img src="../../assets/event-stream-destinations/criteo-connection-settings-2.png" alt="Criteo connection settings" />
+<img src="../../assets/event-stream-destinations/criteo-connection-settings-3.png" alt="Criteo connection settings" />
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/criteo.mdx:34]: As discussed, the images need to be update to reflect the new setting.

File: docs/destinations/streaming-destinations/criteo.mdx (hunk 1 above)
Comment (ItsSudip) [reply docs/destinations/streaming-destinations/criteo.mdx:61]: Yes @ameypv-rudder this looks good.

**Review by ItsSudip — APPROVED** [review]

File: docs/destinations/streaming-destinations/criteo.mdx (hunk 2 above)
Comment (ameypv-rudder) [reply docs/destinations/streaming-destinations/criteo.mdx:34]: I've updated the images.

File: docs/destinations/streaming-destinations/criteo.mdx (hunk 1 above)
Comment (ItsSudip) [reply docs/destinations/streaming-destinations/criteo.mdx:61]: We can use this link.

Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/criteo.mdx:61]:
```suggestion
- **Mapping RudderStack events to the Criteo standard events**: Enter the **Event Name** and the corresponding [**Standard Event**](https://www.criteo.com/wp-content/uploads/2018/09/CSPOneTag_v1.1.pdf) from the dropdown. This standard event will be triggered when the mapped RudderStack event is called.
```

File: docs/destinations/streaming-destinations/criteo.mdx [hunk 3]
```diff
@@ -3,15 +3,15 @@ title: "Criteo"
...
 ---
 
-[**Criteo**](https://www.criteo.com/) is a popular online display advertising platform that offers various solutions to increase your website traffic. It also helps you generate brand awareness and boost sales, conversions, and app installs.
+[Criteo](https://www.criteo.com/) is a popular online display advertising platform that offers various solutions to increase your website traffic, generate brand awareness, and boost sales.
 
 RudderStack supports Criteo as a destination to which you can send your event data directly.
 
 ## Getting started
 
 Before configuring Criteo as a destination in RudderStack, verify if Criteo supports the source platform you are sending the events from. Refer to the following table:
```
Comment (ishachopra0401) [inline docs/destinations/streaming-destinations/criteo.mdx:12]: Before configuring Criteo as a destination in RudderStack, verify if the source platform is supported by Criteo by referring to the table below:

File: docs/destinations/streaming-destinations/criteo.mdx [hunk 4]
```diff
@@ -110,21 +118,19 @@ rudderanalytics.track("Product Viewed", {
 })
 ```
 
-In this snippet, RudderStack captures the information related to the `Product Viewed` event and the associated details such as quantity, price, category, etc.
+In the above example, RudderStack captures the information related to the `Product Viewed` event and the associated details such as quantity, price, category, etc.
 
-The following table details the mapping of the <Link to="/event-spec/ecommerce-events-spec/">RudderStack e-commerce events</Link> and the [**Criteo OneTag events**](https://support.criteo.com/s/article?article=All-Criteo-OneTag-events-and-parameters&language=en_US).
+The following table details the mapping of the <Link to="/event-spec/ecommerce-events-spec/">RudderStack e-commerce events</Link> and the [Criteo OneTag events](https://support.criteo.com/s/article?article=All-Criteo-OneTag-events-and-parameters&language=en_US).
```
Comment (ishachopra0401) [inline docs/destinations/streaming-destinations/criteo.mdx:123]: This criteo link is redirected to its home page. Maybe this should be the one: https://help.criteo.com/kb/guide/en/all-criteo-onetag-events-and-parameters-vZbzbEeY86/Steps/775825

**Review by ishachopra0401 — APPROVED** [review]

File: docs/destinations/streaming-destinations/criteo.mdx (hunk 4 above)
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/criteo.mdx:123]:
```suggestion
The following table details the mapping of the <Link to="/event-spec/ecommerce-events-spec/">RudderStack e-commerce events</Link> and the [Criteo OneTag events](https://help.criteo.com/kb/guide/en/all-criteo-onetag-events-and-parameters-vZbzbEeY86/Steps/775825).
```

File: docs/destinations/streaming-destinations/criteo.mdx (hunk 3 above)
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/criteo.mdx:12]:
```suggestion
Before configuring Criteo as a destination in RudderStack, verify if the source platform is supported by Criteo by referring to the table below:
```

**Pull request merged**